_posts/2016-01-06-format-support.md

```diff
@@ -0,0 +1,115 @@
+   both compressed and uncompressed data, in brightfield and fluorescence.
+
+These points are specific to this format but similar issues occur with other
+proprietary formats. As a team, we are not comfortable with releasing a reader
+implementation that works on a limited set of file format variants, or
+requires time consuming and computationally expensive reprocessing and pyramid
+creation, just because of the implementation choices made by the format
+designers.
+
+##A philosophical point about our funding and resources
+
+The OME Consortium and the wider community has worked steadily since 2002,
+funded mostly by grants from non-profit charities and public funders, to build
+tools for the wider scientific community.
+
+Building readers for proprietary formats has never been funded, and we don’t
+think it would ever be funded by any grant funding panel. New readers are
+created either by diverting our precious resources from other projects, by
+contributions by the community (most recently by the companies themselves) or
+by work commissioned by customers of Glencoe Software.  We certainly listen to
+the community and adjust our priorities based on requests, but we can’t do
+everything with limited resources.
+
+Perhaps we could crowdsource the funding for file formats  but that misses the
+point—the formats we often lack the resources to support are those which are
+complex, expensive, difficult, proprietary, closed formats, designed to lock
+their users into a single, proprietary software application. The community’s
+resources are finite and should be used for things other than reverse
+engineering this type of format; work that, if subjected to peer review, would
+be declined as a waste of community resources.  Several of those “other
+things” were discussed at our most recent Annual Users Meeting and represent
+key technologies that the community needs to achieve its scientific goals.
+
+Over the last few years, we have seen efforts by several commercial imaging
+companies to support open formats, provide open APIs, and to make it easier
+for researchers and clinicians to work with the data acquired by their
+instruments.  We have also received specifications and input from several
+imaging companies, which we have used to improve our own work and output.  We
+applaud this trend; ultimately it means scientists, clinicians, engineers and
+developers spend less time dealing with data formats and more time doing
+science, developing new technologies and treating patients.
+
+##What you can do
+
+The community has the power to change this situation. You are paying for these
+proprietary formats. You can condition your purchase, continued payment of
+support and maintenance fees, etc on:
+
+- the delivery of a rational, well-designed, efficient, open format
+- use of open compression schemes
+- support for the community’s efforts to deliver open readers for these files
```

Comment:
The style guide we use for bulleted lists says no punctuation unless each point is a complete sentence.